### ISSUE TITLE
Incorrect position reported by space-infix-ops

### ISSUE THREAD

**Issue by ilyavolodin** [eslint/eslint#2354]
space-infix-ops always reports the location based on the very first item in boolean expression.

```js
/* eslint space-infix-ops: 2, no-alert: 0 */
/* eslint-env browser */

var test, test2;

var output = test || (test && test.value) ||(test2 && test2.value);

alert(output);
```

reports the following:
```
6:13 - Infix operators must be spaced. (space-infix-ops)
````

I think what should be reported is 6:42 as there are multiple infix operators on this line.

However, when provided with the following code:

```js
/* eslint space-infix-ops: 2, no-alert: 0 */
/* eslint-env browser */

alert(4 + 5*2);
```

it correctly reports:
```
4:10 - Infix operators must be spaced. (space-infix-ops)
```

Tests where done in online demo, so latest version of eslint.

**Comment by nzakas** [comment]
Yeah, we should fix this.
